.github/workflows/release.yaml

```diff
@@ -37,19 +37,15 @@ jobs:
       - name: Create a release candidate
         if: ${{ contains(github.ref, '-rc') }}
         run: |
-          gh release create $TAG --prerelease --title $TAG --notes "Release candidate" ./out/ai-gateway-crd-helm-${TAG}.tgz
-          gh release create $TAG --prerelease --title $TAG --notes "Release candidate" ./out/ai-gateway-helm-${TAG}.tgz
-          gh release upload $TAG ./out/ai-gateway-crd-helm-${TAG}.tgz
-          gh release upload $TAG ./out/ai-gateway-helm-${TAG}.tgz
+          gh release create $TAG --prerelease --title $TAG --notes "Release candidate" ./out/ai-gateway-crd-helm-${TAG}.tgz ./out/ai-gateway-helm-${TAG}.tgz
+          gh release upload $TAG ./out/ai-gateway-crd-helm-${TAG}.tgz ./out/ai-gateway-helm-${TAG}.tgz
```

Comment:
i think `upload` wouldn't be needed but i think harmless so i will go ahead and merge